Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -150,3 +152,24 @@ QVector<QString> Settings::GetProfiles(const InputConfig* config) const
 
   return vec;
 }
+
+const QString Settings::GetInstalledWiiSystemMenuVersion() const
+{
+  const auto& sys_menu_loader = DiscIO::NANDContentManager::Access().GetNANDLoader(
```

Comment:
Please use IOS for this, not NANDContentManager (which accesses tons of other files for no reason and may return wrong results, since it caches info). 

This code should possibly be merged with the already existing instance in https://github.com/spycrab/dolphin/blob/e7623091390226b453ebcd922cfc9ab841125337/Source/Core/DolphinQt2/MenuBar.cpp#L284 (I'm not sure why this is considered parts of "Settings"?)